pages/index.js

```diff
@@ -21,27 +22,84 @@ export default function HomePage() {
     .filter((transaction) => transaction.type === "expense")
     .reduce((acc, transaction) => acc - transaction.amount, 0);
   const balanceTotal = incomeTotal - expenseTotal;
-  console.log(balanceTotal);
 
   return (
     <>
-     <StyledAccountBalance balance={balanceTotal}>
+      <StyledAccountBalance balance={balanceTotal}>
```

Comment:
Your AccountBalance shoule be a distinct Component (also add the income and expense logic to that component. Then call the component here instead and pass the transactions to it.